src/utils/color-utils.ts

```diff
@@ -19,16 +28,16 @@ export const interpolateChannels = (
 export const interpolateColor = (
   [origR, origG, origB]: RgbColor,
   [newR, newG, newB]: RgbColor,
-  steps: number
-): RgbColor[] => {
-  return new Array(steps)
+  steps: number,
+  easingFunction = Easing.Linear
+): RgbColor[] =>
+  new Array(steps)
     .fill(0)
     .map((_, i) => [
-      origR + ((i + 1) * (newR - origR)) / steps,
-      origG + ((i + 1) * (newG - origG)) / steps,
-      origB + ((i + 1) * (newB - origB)) / steps,
+      easingFunction(origR + ((i + 1) * (newR - origR)) / steps),
+      easingFunction(origG + ((i + 1) * (newG - origG)) / steps),
+      easingFunction(origB + ((i + 1) * (newB - origB)) / steps),
```

Comment:
`easingFunction` takes in numbers from `0.0` to `1.0` right? I think we need to use the `i` for that
```suggestion
      easingFunction(i / steps) * (origR + ((i + 1) * (newR - origR)) / steps),
      easingFunction(i / steps) * (origG + ((i + 1) * (newG - origG)) / steps),
      easingFunction(i / steps) * (origB + ((i + 1) * (newB - origB)) / steps),
```

Reply:
TRUE!